packages/code-du-travail-data/indexing/vectorizer.js

```diff
@@ -0,0 +1,70 @@
+// import fetch from "node-fetch";
+// import semantic_stopwords from "../dataset/stop_words";
+const fetch = require("node-fetch");
+const semantic_stopwords = require("../dataset/stop_words");
```

Comment:
Did you switch from ES modules to CommonJS for a reason ?

Reply:
code-du-travail-api is using CommonJS, and throwing an exception when requiring code-du-travail-data ES module, will see if I can find a work around